docs/content/documentation/templates/pagination.md

```diff
@@ -52,6 +52,13 @@ A paginated taxonomy gets two variables aside from the `paginator` variable:
 
 See the [taxonomies page](@/documentation/templates/taxonomies.md) for a detailed version of the types.
 
+## SEO
+
+It is preferable to not include paginated pages in sitemap since they are non-canonical pages as explained
```

Comment:
I would avoid making a call like that since it seems there are a lot of opinions: https://www.reddit.com/r/TechSEO/comments/10gfiob/should_pagination_pages_be_included_in_your/
Just describe the setting behaviour